release_notes/ob-release-notes.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ob-release-notes"]
+= Release notes
+include::_attributes/common-attributes.adoc[]
+:context: ob-release-notes
+
+toc::[]
+
+Release notes contain information about new and deprecated features, breaking changes, and known issues. The following release notes apply for the most recent {builds-shortname} releases on {ocp-product-title}.
+
+{builds-shortname} is an extensible build framework based on the link:https://shipwright.io/[Shipwright project], which you can use to build container images on an {ocp-product-title} cluster. You can build container images from source code and Dockerfiles by using image build tools, such as Source-to-Image (S2I) and Buildah. You can create and apply build resources, view logs of build runs, and manage builds in your {ocp-product-title} namespaces.
+
+{builds-shortname} includes the following capabilities:
+
+* Standard Kubernetes-native API for building container images from source code and Dockerfiles
+* Support for Source-to-Image (S2I) and Buildah build strategies
+* Extensibility with your own custom build strategies
+* Execution of builds from source code in a local directory
+* Shipwright CLI for creating and viewing logs, and managing builds on the cluster
+* Integrated user experience with the *Developer* perspective of the {ocp-product-title} web console
+
+For more information about {builds-shortname}, see xref:../about/overview-openshift-builds.adoc#overview-openshift-builds[Overview of {builds-shortname}].
+
+
+include::modules/ob-compatibility-support-matrix.adoc[leveloffset=+1]
+
+include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
+
+// Modules included, most to least recent
+
+include::modules/ob-release-notes-1-1.adoc[leveloffset=+1]
```

Comment:
I am not sure based on the changes I see in this PR, so just to make sure:
* include the `ob-release-notes-1-2.adoc` file in the `modules` folder (module)
* include the  `ob-release-notes.adoc` file in the `release_notes` folder (assembly)